torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14159,6 +14159,32 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
             # RuntimeError: attribute lookup is not defined on builtin
             DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),
         )),
+    UnaryUfuncInfo(
+        'cdouble',
+        op=lambda x, *args, **kwargs: x.cdouble(*args, **kwargs),
```

Comment:
Can we just write `op=Tensor.cdouble`? I think the answer is no, but I don't remember the reason off the top of my head.